pkg/kms/service.go

```diff
@@ -110,6 +112,26 @@ func (s *service) Verify(keystoreID, keyID string, sig, msg []byte) error {
 	return err
 }
 
+// Encrypt encrypts a message and aad.
```

Comment:
Is `aad` encrypted? I'm not exactly sure if this is applicable or not, but the JWE spec says that it's not... check with @Baha-sk and see if that function description is accurate

Reply:
AD stands for "associated data" and is bind to the ciphertext in a sense that you should provide it when decrypting the value, otherwise the decryption fails with an authenticity error. That method description comes from the Aries framework.

Reply:
@aholovko can you link me to that method description? I'm still unsure about whether aad is encrypted (which is what your method description says)

Reply:
sure, it's here https://github.com/hyperledger/aries-framework-go/blob/80cb211c6dd6e5af44b9cfb2204a58979d70c2c0/pkg/crypto/api.go#L14

Reply:
Probably "Encrypt encrypts a message **with** aad" would be a more precise description. 🤔

Reply:
@aholovko yeah, I think that's probably the more accurate description

Reply:
Once that's changed I think the PR is good to go!

Reply:
@DRK3, PR was already merged so I would change it in my next PR.